manifests/init.pp

```diff
@@ -14,10 +14,13 @@
     ensure => 'directory',
   }
 
-  class { 'composer':
+  class { '::composer':
```

Comment:
Probably want to add an option to tell it to skip the dep so we can avoid conflicts with other things depending on composer